2024-01-30 16:33:24 - ERROR - /private/var/folders/k1/23kw474556dc5nt9ghkdmqfm848fbh/T/tmpdm09yfq8/.temp/https:__github.com_example_repo.git
2024-01-30 16:33:24 - ERROR - Traceback (most recent call last):
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/coupling/logical_coupling.py", line 386, in run
    util.pull(path_to_cloned_repo, branch, logger)
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/coupling/util.py", line 42, in pull
    repo = Repo(path_to_repo)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/.venv/lib/python3.11/site-packages/git/repo/base.py", line 215, in __init__
    raise NoSuchPathError(epath)
git.exc.NoSuchPathError: /private/var/folders/k1/23kw474556dc5nt9ghkdmqfm848fbh/T/tmpdm09yfq8/.temp/https:__github.com_example_repo.git

2024-01-30 16:33:24 - ERROR - /private/var/folders/k1/23kw474556dc5nt9ghkdmqfm848fbh/T/tmp2_n6ve31/.temp/https:__github.com_example_repo.git
2024-01-30 16:33:24 - ERROR - Traceback (most recent call last):
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/coupling/logical_coupling.py", line 386, in run
    util.pull(path_to_cloned_repo, branch, logger)
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/coupling/util.py", line 42, in pull
    repo = Repo(path_to_repo)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/.venv/lib/python3.11/site-packages/git/repo/base.py", line 215, in __init__
    raise NoSuchPathError(epath)
git.exc.NoSuchPathError: /private/var/folders/k1/23kw474556dc5nt9ghkdmqfm848fbh/T/tmp2_n6ve31/.temp/https:__github.com_example_repo.git

2024-01-30 16:34:22 - ERROR - /private/var/folders/k1/23kw474556dc5nt9ghkdmqfm848fbh/T/tmppk02dln6/.temp/https:__github.com_example_repo.git
2024-01-30 16:34:22 - ERROR - Traceback (most recent call last):
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/coupling/logical_coupling.py", line 386, in run
    util.pull(path_to_cloned_repo, branch, logger)
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/coupling/util.py", line 42, in pull
    repo = Repo(path_to_repo)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/ncdaam/PycharmProjects/LogicalCouplingTool/.venv/lib/python3.11/site-packages/git/repo/base.py", line 215, in __init__
    raise NoSuchPathError(epath)
git.exc.NoSuchPathError: /private/var/folders/k1/23kw474556dc5nt9ghkdmqfm848fbh/T/tmppk02dln6/.temp/https:__github.com_example_repo.git

